src/transformers/models/gpt_bigcode/modeling_gpt_bigcode.py

```diff
@@ -54,15 +50,14 @@
 
 GPT_BIGCODE_PRETRAINED_MODEL_ARCHIVE_LIST = [
     "bigcode/santacoder-fast-inference",
-    # See all GPTBigCode models at https://huggingface.co/models?filter=gpt_bigcode
+    # See all GPTBigCode models at https://huggingface.co/models?filter=santacoder-
```

Comment:
Not sure about this...

Reply:
Made a new model at `bigcode/gpt_bigcode-santacoder` with some minor changes for the HF version, so we can go back to the better filter.